docs/source/collect/certificates.md

```diff
@@ -0,0 +1,115 @@
+---
+title: TLS Certificates
+description: Collect Certificate Chain information from Kubernetes ConfigMaps and Secrets in collected output
+---
+
+The `certificates` collector can be used to gather information about the TLS certificates from Kubernetes ConfigMaps and Secrets. This collector can be used multiple times, referencing different Secrets and ConfigMaps.
+
+## Parameters
+
+In addition to the [shared collector properties](https://troubleshoot.sh/docs/collect/collectors/#shared-properties), the `certificates` collector accepts the following parameters:
+
+##### `name` (Required)
+
+The name of the ConfigMap or Secret.
+
+##### `namespace` (Required)
+
+The namespaces where the Secret or ConfigMap exists. If multiple namespaces are specified, resources that match one of the namespaces are collected.
+> Example:
+```yaml
+collectors:
+  - certificates: 
+    secrets:
+      - name: envoycert
+        namespaces:
+          - kube-system
+          - projectcontour
+```
```

Comment:
Recommend deleting this example (lines 19 - 28) as it's already provided in the Example Collector Definition below. As is, it looked a little noisy in the preview

Reply:
removed